deploy-deps.sh

```diff
@@ -124,6 +132,11 @@ deploy_registry() {
     kubectl apply -k "${script_path}/dependencies/registry"
     retry "kubectl wait --for=condition=Ready --timeout=240s -n kind-registry -l run=registry pod" \
           "The local registry did not become available within the allocated time"
+    # Copy the registry secret to cert-manager ns so the ca cert can be distrubuted
+    kubectl delete secret -n cert-manager --ignore-not-found=true local-registry-tls
+    kubectl get secret local-registry-tls --namespace=kind-registry -o yaml \
+        | grep -v '^\s*namespace:\s' | kubectl apply --namespace=cert-manager -f -
```

Comment:
why grep is needed here?

Reply:
It's gerping everything but the namepsace line, and then we apply the same manifest to another namespace.